examples/bert/notebook/multi_ep_search.ipynb

```diff
@@ -0,0 +1,400 @@
+      "|                                                                                    |                                                                                    |                             |                |   \"latency-min\": 12.20827                  |\n",
+      "|                                                                                    |                                                                                    |                             |                | }                                          |\n",
+      "+------------------------------------------------------------------------------------+------------------------------------------------------------------------------------+-----------------------------+----------------+--------------------------------------------+\n",
+      "| 0_OnnxConversion-53fc6781998a4624b61959bb064622ce-bacd09aac459011bdd7b4c86b291808c | 53fc6781998a4624b61959bb064622ce                                                   | OnnxConversion              |       11.6793  |                                            |\n",
+      "+------------------------------------------------------------------------------------+------------------------------------------------------------------------------------+-----------------------------+----------------+--------------------------------------------+\n",
+      "| 7_OrtTransformersOptimization-0-ce57cc2e4971809c50a213199be68734-gpu-tensorrt      | 0_OnnxConversion-53fc6781998a4624b61959bb064622ce-bacd09aac459011bdd7b4c86b291808c | OrtTransformersOptimization |        8.23434 |                                            |\n",
+      "+------------------------------------------------------------------------------------+------------------------------------------------------------------------------------+-----------------------------+----------------+--------------------------------------------+\n",
+      "[2024-04-18 17:28:42,448] [INFO] [engine.py:285:run] Package top ranked 0 models as artifacts\n",
+      "[2024-04-18 17:28:42,448] [WARNING] [packaging_generator.py:48:generate_output_artifacts] No model is selected. Skip packaging output artifacts.\n"
+     ]
+    }
+   ],
+   "source": [
+    "! python -m olive.workflows.run --config bert_auto_opt_gpu.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Based on the above olive running history, we can see that:\n",
+    "1. Olive searched the configs for several rounds and found the optimal optimization and packed it with zip format to save.\n",
+    "2. During the search process, we can see that the invalid config are pruned and the valid configs are evaluated.\n",
+    "3. The best config is saved in the `models/bert_gpu` folder. And here is comparison between the output model and input model.\n",
```

Comment:
Does the best config clarify which EP produced optimal output model?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/8a4624c0-8208-4e63-a38b-d29fcff3ed9c)
Yes, it has that.